BUILD.md

```diff
@@ -249,3 +249,12 @@ Rust components:
 **Note:** The `run_tests` command provides the ``-x`` flag to allow you to exclude
 various components from the tests. You can also specify which tests to run
 with the ``-m`` flag. Run the command `run_tests -h` for help.
+
+License
```

Comment:
This is already in README.md. This particular file (BUILD.md) should probably be removed (content moved to sphinx-doc so it gets published to the website), but certainly not duplicate the README.md.